cypress/e2e/browsing.cy.js

```diff
@@ -1,12 +1,15 @@
-describe('The home page', () => {
-  it('should be able to reach the browse page', () => {
+describe('Browsing', () => {
+  it('a user can search from the home page and be directed to "/browse" with a blank query', () => {
     // Start from the home/index page
-    cy.visit('http://localhost:3000/')
+    cy.visit('/')
 
     // Find the search button and perform an empty search, which should lead to the browse page
     cy.get('button.search-button').click()
 
     // The new url should include "/browse"
     cy.url().should('include', '/browse')
+
```

Comment:
can we add another test here that the url does _not_ include a query term?

Reply:
wait i think I misread this. The original test did not include a query term, it leaves the search bar blank.

I did add another test that DOES include a query term as well